server/utils/__init__.py

```
@@ -0,0 +1,8 @@
+def string_to_boolean(s):
+    if s.lower() == "true":
+        return True
+    elif s.lower() == "false":
+        return False
+    else:
+        # Handle other cases, e.g., raise an error, return None, or a default boolean
+        raise ValueError(f"Invalid boolean string: {s}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)